group/edwards25519/suite.go

```diff
@@ -9,24 +9,24 @@ import (
 	"github.com/dedis/fixbuf"
 
 	"github.com/dedis/kyber"
-	"github.com/dedis/kyber/cipher/sha3"
 	"github.com/dedis/kyber/group/internal/marshalling"
+	"github.com/dedis/kyber/xof/blake"
 )
 
-// SuiteEd25519 implements some basic functionalities such as Group, HashFactory
-// and CipherFactory.
+// SuiteEd25519 implements some basic functionalities such as Group, HashFactory,
+// CipherFactory and XOFFactory.
 type SuiteEd25519 struct {
 	Curve
 }
 
-// Hash return a newly instanciated sha256 hash function
+// Hash return a newly instanciated sha256 hash function.
 func (s *SuiteEd25519) Hash() hash.Hash {
 	return sha256.New()
 }
 
-// Cipher returns the SHA3/SHAKE128 Sponge Cipher
-func (s *SuiteEd25519) Cipher(key []byte, options ...interface{}) kyber.Cipher {
-	return sha3.NewShakeCipher128(key, options...)
+// XOF returns an XOF which is implemented via the Blake2b hash.
```

Comment:
Do you pronounce `Ksof` or `X.O.F.`? Just for the `an`/`a`...

Reply:
I say ex.oh.ef in my head. Thus the "an".

Reply:
OK, ex.oh.ef in the head and ksof when speaking - I wonder what that means for usage of `an`/`a` ;)